client/state/sites/hooks/test/use-selected-site-selector.js

```diff
@@ -0,0 +1,49 @@
+import { FEATURE_WP_SUBDOMAIN } from '@automattic/calypso-products';
+import * as redux from 'react-redux';
+import hasAvailableSiteFeature from 'calypso/state/selectors/has-available-site-feature';
+import { useSelectedSiteSelector } from '../';
+import { isJetpackSite } from '../../selectors';
+
+const useSelector = jest.spyOn( redux, 'useSelector' );
+
+describe( 'useSelectedSiteSelector()', () => {
+	test( 'executes selector with selected site id passed in', () => {
+		const state = {
+			sites: {
+				items: {
+					123: {
+						jetpack: true,
+					},
+				},
+			},
+			ui: {
+				selectedSiteId: 123,
+			},
+		};
+		useSelector.mockImplementation( ( selector ) => selector( state ) );
+		expect( useSelectedSiteSelector( isJetpackSite ) ).toEqual( true );
```

Comment:
nit: `.toBe( true )` is a bit better. `.toEqual` does a deep comparison of objects, and we don't need this extra power here.

```suggestion
		expect( useSelectedSiteSelector( isJetpackSite ) ).toBe( true );
```

Reply:
Hijacked your comment to add suggestion, thanks!